www/md_base/src/app/common/directives/panel/panel.directive.coffee

```diff
@@ -8,10 +8,7 @@ class Panel extends Directive
             controllerAs: 'panel'
             bindToController: true
             scope:
-                name: '='
-                title: '='
-                isCollapsed: '='
-                template: '='
+                bind: '='
```

Comment:
I think current interface is simpler. It makes no sense that we separate attributes of the panel. Although the internal implementation will become a little complex, the directive's interface will be easier to use for the outsiders.